lib/src/button/Button.tsx

```diff
@@ -31,22 +44,14 @@ const DxcButton = ({
 
   return (
     <ThemeProvider theme={colorsTheme.button}>
-      <DxCButton
-        type={type}
-        margin={margin}
-        mode={mode !== "primary" && mode !== "secondary" && mode !== "text" ? "primary" : mode}
-        disabled={disabled}
-        iconPosition={iconPosition}
-        size={size}
-        backgroundType={backgroundType}
-        icon={icon}
-      >
+      <ButtonContainer margin={margin} size={size} backgroundType={backgroundType}>
```

Comment:
I don't think this container is longer necessary since the margin and the size can be applied directly to the button.

Reply:
Done!

Reply:
Done!